templates/base/storybook/.storybook/main.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  stories: ['../../src/components/**/**/*.stories.mdx'],
+  stories: ['../../src/components/**/**/*.(stories|story).(js|mdx)'],
```

Comment:
Lets update the generator to use *.story.js storybook input file, instead of MDX.

Reply:
changed to story.js